docker-compose.yml

```diff
@@ -22,10 +22,12 @@ services:
     ports:
       - "5555:5432"
   redis:
-    image: redis
+    image: redis:alpine
     container_name: redis
+    hostname: redis
     ports:
-       - "127.0.0.1:16379:6379"
+       - "16379:6379"
```

Comment:
why do you need outer port `16379`?

Reply:
There was an exception that port  6379 is already in use, so i used that one